monitors/corrymonitor/README_CorryMonitor.md

```diff
@@ -0,0 +1,103 @@
+# CorryMonitor
+
+The `CorryMonitor` is an online monitoring tool for the use at test beams. It was designed to be versatile such that it can be used by a large user-base. See the [AIDAinnova Milestone 10 Report](https://zenodo.org/records/8402178) and [Deliverable 3.4 Report](https://zenodo.org/records/12731423) for a write-up of the `CorryMonitor` working principle and features. This document is meant to guide users through the example provided for the `CorryMonitor` in this repository.
+The most important take-away is that this monitor utilises the existing test beam data analysis program `corryvreckan` for displaying of online monitoring plots.
+
+## Prerequisites
+
+Since `CorryMonitor` relies on the widely used test beam reconstruction and analysis software `corryvreckan` for plotting of the data, it goes without saying that a local installation of `corryvreckan` is required. `corryvreckan` itself has minimal external dependencies, requiring only a `ROOT` installation. Please see the documentation on the [project website](https://project-corryvreckan.web.cern.ch/project-corryvreckan/page/about/) for details and user guides.
+**Please note that for the `corryvreckan` installation the `-DBUILD_EventLoaderEUDAQ2=ON` flag needs to be set to work with `CorryMonitor`.**
+
+## Installation
+
+To build the `CorryMonitor`, the `EUDAQ_BUILD_CORRYMONITOR` needs to be set to `ON` (default value is `OFF`).
+
+## Known Restrictions of the `CorryMonitor`
+
+- The CorryMonitor is currently exclusively running on Linux due to the use of *inotify*.
+- It is currently required to run the Monitor on the same machine as the RunControl to allow access to the `eudaq` configuration files. (The DataCollectors can run remotely via XRootD, explained below).
```

Comment:
Is this correct? The config and init file sections should be read by the runcontroller and transmitted through the usual mechanism. Are you accessing them directly?

Reply:
The problem is not the initialisation and configuration of the monitor, this gets passed as usual via the runcontrol.
The problem is the configuration for the datacollectors, because I need to get the file-naming pattern from there. But there is currently no mechanism to send the DataCollector-configuration to a Monitor module, I believe. So this where the limitation comes from. If it is not on the same machine, the CorryMonitor cannot access the information from the DataCollector configuration